hscode.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{A HTTP status code lookup command line tool.}
   spec.description   = %q{Quickly look up any status code without leaving your terminal.}
-  spec.homepage      = ''
+  spec.homepage      = 'https://github.com/akabiru/hscode'
```

Comment:
@Habu-Kagumba Should we change this to the docs page? https://akabiru.github.io/hscode/